scripts/quick_start.py

```diff
@@ -8,26 +8,14 @@
 import torch
 
 from torchao.quantization import Int4WeightOnlyConfig, quantize_
+from torchao.testing.model_architectures import ToyTwoLinearModel
 from torchao.utils import benchmark_model
 
 # ================
 # | Set up model |
 # ================
 
-
-class ToyLinearModel(torch.nn.Module):
-    def __init__(self, m: int, n: int, k: int):
-        super().__init__()
-        self.linear1 = torch.nn.Linear(m, n, bias=False)
-        self.linear2 = torch.nn.Linear(n, k, bias=False)
-
-    def forward(self, x):
-        x = self.linear1(x)
-        x = self.linear2(x)
-        return x
-
-
-model = ToyLinearModel(1024, 1024, 1024).eval().to(torch.bfloat16).to("cuda")
+model = ToyTwoLinearModel(1024, 1024, 1024).eval().to(torch.bfloat16).to("cuda")
```

Comment:
this should be reverted as well I think, since it seems like to be a copy of the quick_start.rst